example_komunikaty.py

```
@@ -0,0 +1,56 @@
+#-----------------------------------------------------------------------------
+#jak to ma wygladac w glownym kodzie:
+#w każdej strukturze mają znaleźć się elementy z poniższych komentarzy  
+class Robot:
+    def __init__(self, status_log):
+        self.status_log = status_log    #do init dodajecie ten wiersz
+        self.is_moving = False
+        self.endstop_floor_1 = FakeEndstop()
```

Ruff (0.11.9)

Undefined name `FakeEndstop`

(F821)